modules/delegates.js

```diff
@@ -687,17 +652,19 @@ Delegates.prototype.updateForgingStatus = function(
  * Gets delegate list based on input function by vote and changes order.
  *
  * @param {number} round
- * @param {function} source - Source function for get delegates
  * @param {function} cb - Callback function
  * @param {Object} tx - Database transaction/task object
  * @returns {setImmediateCallback} cb, err, truncated delegate list
  * @todo Add description for the params
  */
-Delegates.prototype.generateDelegateList = function(round, source, cb, tx) {
+Delegates.prototype.getForgersList = function(round, cb, tx) {
```

Comment:
I don't like the name, the function is sometimes (when cache not available) really generating the list and not just getting.